qdrant-landing/content/articles/fastembed.md

```diff
@@ -0,0 +1,217 @@
+
+```
+pip install 'qdrant-client[fastembed]'
+```
+
+### Initializing the Qdrant Client
+
+After successful installation, the next step involves initializing the Qdrant Client. This can be done either in-memory or by specifying a database path:
+
+```
+from qdrant_client import QdrantClient# Initialize the clientclient = QdrantClient(":memory:")  # or QdrantClient(path="path/to/db")
+```
+
+### Preparing Documents, Metadata, and IDs
+
+Once the client is initialized, prepare the text documents you wish to embed, along with any associated metadata and unique IDs:
+
+```
+docs = ["Qdrant has Langchain integrations", "Qdrant also has Llama Index integrations"]
+metadata = [{"source": "Langchain-docs"},    {"source": "LlamaIndex-docs"},]
+ids = [42, 2]
+```
+
+Note that the add method we’ll use is overloaded: If you skip the ids, we’ll generate those for you. metadata is obviously optional. So, you can simply use this too:
+
+```
+docs = ["Qdrant has Langchain integrations", "Qdrant also has Llama Index integrations"]
+```
+
+### Adding Documents to a Collection
+
+With your documents, metadata, and IDs ready, you can proceed to add these to a specified collection within Qdrant using the add method:
+
+```
+client.add(
+    collection_name="demo_collection",
+    documents=docs,
+    metadata=metadata,
+    ids=ids
+)
+```
+
+Behind the scenes, Qdrant is using FastEmbed to make the text embedding, generate ids if they’re missing and then adding them to the index with metadata.
+
+INDEX TIME: Sequence Diagram for Qdrant and FastEmbed
+
+### Performing Queries
+
+Finally, you can perform queries on your stored documents. Qdrant offers a robust querying capability, and the query results can be easily retrieved as follows:
+
+```
+search_result = client.query(
+    collection_name="demo_collection",
+    query_text="This is a query document"
+)
+print(search_result)
+```
+
+Behind the scenes, we first convert the query_text to the embedding and use that to query the vector index.
+
+QUERY TIME: Sequence Diagram for Qdrant and FastEmbed integration
+
+By following these steps, you effectively utilize the combined capabilities of FastEmbed and Qdrant, thereby streamlining your embedding generation and retrieval tasks.
+
+Qdrant is designed to handle large-scale datasets with billions of data points. Its architecture employs techniques like binary and scalar quantization for efficient storage and retrieval. When you inject FastEmbed’s CPU-first design and lightweight nature into this equation, you end up with a system that can scale seamlessly while maintaining low latency.
+
+## Summary
+
+If you're curious about how FastEmbed and Qdrant can make your search tasks a breeze, why not take it for a spin? You get a real feel for what it can do. Here are two easy ways to get started:
+
+1. **Cloud Trial**: You can activate a Cloud trial and see how it fits your needs. It's a click away: [Qdrant Cloud](https://cloud.qdrant.io).
```

Comment:
We should not use the exact specifications of our free plan since it is not constant. On GCP we already have a different offering and it will also change in the future.